source/Personality.h

```diff
@@ -63,6 +63,9 @@ class Personality {
 	bool IsMining() const;
 	bool Harvests() const;
 	bool IsSwarming() const;
+	bool IsRoving() const;
+	bool IsCloaking() const;
+	bool IsBored() const;
```

Comment:
IsBored() should be at the same location in the cpp as here; but its after IsEvasive

Reply:
Maybe I was not clear: the methods should be in the same place in the cpp as in the hpp
Isbored should just be after isevasive here, as for the cpp (except if you changed the order in the cpp)